app/services/kyc_service.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class KycService
+  # passport front + selfie OR driver license front and back + selfie IR id card front and back + selfie
+  REQUIRED_DOC_AMOUNT = { 'PASSPORT': 2, 'DRIVERS_LICENSE': 3, 'GOVERNMENT_ID': 3 }.freeze
```

Comment:
For passport we need front + selfie OR driver license front and back + selfie OR id card front and back + selfie